.github/workflows/chart-release.yaml

```diff
@@ -1,23 +1,3 @@
-#################################################################################
```

Comment:
Copyright header needs to stay. You can read about this in the [Eclipse Foundation Project Handbook](https://www.eclipse.org/projects/handbook/#ip-copyright-headers).
Also, if the file is changed, check the years are up to date